br/br-auto-tune.md

```diff
@@ -0,0 +1,89 @@
+
+Auto-tune is a coarse-grained solution for limiting backup speed. It reduces the need for manual tuning. However, because of the lack of fine-grained control, auto-tune might not be able to completely remove the impact of backup on the cluster.
+
+The auto-tune feature has the following issues and corresponding solutions:
+
+- Issue 1: For **write-heavy clusters**, auto-tune might put the workload and backup tasks into a "positive feedback loop": the backup tasks take up too many resources, which causes the cluster to use fewer resources; at this point, auto-tune might mistakenly assume that the cluster is not under heavy workload and thus allow BR to run faster. In such cases, auto-tune is ineffective.
+
+    - Solution: Manually adjust `backup.num-threads` to a smaller number to limit the number of threads used by backup tasks. The working principle is as follows:
+
+        The backup process includes lots of SST decoding, encoding, compression, and decompression, which consume CPU resources. In addition, previous test cases have shown that during the backup process, the CPU utilization of the thread pool used for backup is close to 100%. This means that the backup tasks take up a lot of CPU resources. By adjusting the number of threads used by the backup tasks, TiKV can limit the CPU cores used by backup tasks, thus reducing the impact of backup tasks on the cluster performance.
+
+- Issue 2: For **clusters with hotspots**, backup tasks on the TiKV node that has hotspots might be excessively limited, which slows down the overall backup process.
+
+    - Solution: Eliminate the hotspot node, or disable auto-tune on the hotspot node (this might reduce the cluster performance).
+
+- Issue 3: For scenarios with **high traffic jitter**, because auto-tune adjusts the speed limit on a fixed interval (1 minute by default), it might not be able to handle high traffic jitter. For details, see [`auto-tune-refresh-interval`](#implementation).
+
+    - Solution: Diable auto-tune.
+
+## Implementation
+
+Auto-tune adjusts the size of the thread pool used by BR backup tasks to ensure that the overall CPU utilization of the cluster does not exceed a specific threshold.
+
+This feature has two related configuration items not listed in the TiKV configuration file. These two configuration items are only for internal tuning. You do not need to configure these two configuration items when you perform backup tasks.
+
+- `backup.auto-tune-remain-threads`:
+
+    - Auto-tune controls the resources used by the backup tasks and ensures that at least `backup.auto-tune-remain-threads` cores are available for other tasks on the same node.
+    - The default value: `round(0.2 * vCPU)`.
+    - Unit: core.
+
+- `backup.auto-tune-refresh-interval`:
+
+    - Every `backup.auto-tune-refresh-interval` minute(s), auto-tune refreshes the statistics and recalculates the maximum number of CPU cores that backup tasks can use.
+    - The default value: `1`.
+    - Unit: minute.
```

Comment:
It is a [`ReadableDuration`](https://github.com/tikv/tikv/blob/5f0e9aeb036bbf3d0cb7213fa46da7a70a75faa1/src/config.rs#L2259) and seems other documents prefer to don't mention the unit for those types.

<img width="786" alt="image" src="https://user-images.githubusercontent.com/36239017/150266556-711cfdc5-d231-44d9-a0ce-444b242e50d2.png">